test/regressions/fixtures/Button/DisabledButtons.js

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+
+export default function Demo() {
+  return (
+    <React.Fragment>
+      <Button disabled style={{ pointerEvents: 'all' }}>
+        disabled with pointer-events
+      </Button>
+      <Button disabled>disabled without pointer-events</Button>
+      <Button aria-disabled>aria-disabled</Button>
```

Comment:
Adding for future reference. I'm inclined to special case `aria-disabled` to act as "disabled but focuseable" for a11y: https://css-tricks.com/making-disabled-buttons-more-inclusive/